modules/01-filter.js

```diff
@@ -169,7 +169,7 @@ async function processCardAction(interaction) {
     const mod = interaction.member,
       embed = u.embed(flag.embeds[0]),
       infraction = await Module.db.infraction.getByFlag(flag);
-
+    if (mod.id == infraction.discordId) return interaction.reply({ content: "You can't handle your own flag!", ephemeral: true });
```

Comment:
Returning here without removing the card from `processing` will lock all mods from taking action on the card until an Icarus reboot.

Reply:
Fixed in latest commit